job_templates/sag_pt_executor/config_fed_client.conf

```diff
@@ -0,0 +1,26 @@
+format_version = 2
+executors = [
+  {
+    # tasks that the defined Executor will support
+    tasks = [
+      # pre-train task
+      "get_weights",
+      # training task
+      "train",
+      # cross-site validation tasks
+      "submit_model",
+      "validate",
+    ]
+    executor {
+      path = "fl.executor.CIFAR10Executor"
```

Comment:
if we copy this into custom (by using job cli), we can use `path = "executor.CIFAR10Executor"`